tools/webpack/configs/common.mjs

```diff
@@ -47,7 +47,7 @@ export default (env, asyncChunkName) => {
 
         return env.SUBVERSION === 'PROD' ? `[name]${env.PATH_VERSION}.js` : '[name].js'
       },
-      chunkFilename: env.SUBVERSION === 'PROD' ? `[name].[chunkhash:8]${env.PATH_VERSION}.min.js` : `[name]${env.PATH_VERSION}.min.js`,
+      chunkFilename: `[name]${env.PATH_VERSION}.min.js`,
```

Comment:
Just want to double check here,  is there any chance that our internal deployments, which often share the same version number, will all try to point at the same async chunk since there is no hash differentiation?

ie does 1.240.0-experiment-1 behind the scenes still just point at nr-spa-1.240.0, same as regular 1.240.0?

Reply:
No. Experiments are contained in a sub-directory of the S3 bucket `/experiments/<branch-name>`. We only ever have one version of the agent in the `/dev/` and `/test/` sub-directories.